Sources/BrowserServicesKit/SecureVault/SecureVaultDatabaseProvider.swift

```diff
@@ -127,10 +135,7 @@ final class DefaultDatabaseProvider: SecureVaultDatabaseProvider {
                 .fetchAll($0)
         }
     }
-
-    /// To be removed once macOS has been updated to use subdomain matching as per
-    /// https://app.asana.com/0/1203822806345703/1204132671693421/f
-    @available(*, deprecated, message: "use websiteAccountsForTopLevelDomain instead")
```

Comment:
I cleaned up this deprecation warning.  We're still using this method from different places and in many cases it would be useful to obtain specific matches